### DISCUSSION TITLE
I can't have the student pack for a bad reason and i don't know how to get it ?

### DISCUSSION THREAD

**Discussion by xstrickers** [community/community#142750] [GitHub Education]
### Hi everyone,

![image](https://github.com/user-attachments/assets/a85da190-305a-45dd-ba37-a0538491c477)

My problem is that i'm a student and my school is eligible my teamates have it but for me it put that reason and i don't know what can i do. As you can see my informations are the same...
![image](https://github.com/user-attachments/assets/b5208165-606c-4eb5-8f6d-2156543bfebd)

![image](https://github.com/user-attachments/assets/ea880b8b-f83b-42d2-ba5b-ff01c6714de6)

**Comment by Inmoresentum** [comment]
If you have a student id card (English), use that instead. IDK but seems like the automated system is not parsing the doc properly!

**Reply by xstrickers** [reply]
I try but it put the same email that i have to check my biling informations
